frontend/src/components/CopyURL.js

```diff
@@ -1,32 +1,31 @@
 import React, { useState } from 'react'
-import { getCurrentURL } from '../utils'
+import PropTypes from 'prop-types'
 import CopyButton from './CopyButton'
 import { Tooltip } from 'react-tippy'
 
-const CopyPostURL = ({ postId }) => {
+const CopyPostURL = ({ post }) => {
   const [buttonText, setButtonText] = useState('Click to copy link')
 
   const copyURL = () => {
-    let currentURL = getCurrentURL()
-
-    if (postId) {
-      currentURL = currentURL + `posts/${postId}`
+    let url = window.location.origin
+    if (post) {
```

Comment:
I think you can use just `window.location.href` instead of `window.location.origin`.

```suggestion
  const CopyPostURL = () => {
  const [buttonText, setButtonText] = useState('Click to copy link');

  const copyUrl = () => {
    navigator.clipboard.writeText(window.location.href)

   setButtonText('Copied!');
    setTimeout(() => {
      setButtonText('Click to copy link')
    }, 1000)
  }

  return (
    <Tooltip
      className="ToolTip"
      arrow
      delay={150}
      duration={1000}
      html={<div>{buttonText}</div>}
    >
      <CopyButton handleClick={copyUrl} text={'Share'} />
    </Tooltip>
  )
}
```

Reply:
@chalecki , It can't be `window.location.href` because we don't need to copy the current page path, we need the path of specific post. See details of the issue (scenarioB describe effet of using 'href'): [#issues/100](https://github.com/Selleo/til/issues/100)

Reply:
@monika1127 Ok it makes sense. There was no description provided in the PR.
What you could do is - instead of passing `post` pass `id` and `slug` 

```js
<CopyPostURL id={post.id} slug={post.slug} />
```

and create the correct path later on in the component (as you did before)

```js
const copyUrl = () => {
    navigator.clipboard.writeText(`${window.location.origin}/post/{id}-{slug}`)

   setButtonText('Copied!');
    setTimeout(() => {
      setButtonText('Click to copy link')
    }, 1000)
  }
```

Reply:
👍